### ISSUE TITLE
Hint should follow question

### ISSUE THREAD

**Issue by pwim** [SachaG/tokyodev#67]
<img width="523" alt="Screenshot 2023-10-19 at 15 47 28" src="https://github.com/SachaG/tokyodev/assets/30756/10e725b3-4dac-48fe-84d8-3acfb5015699">

> (?) What is your country or region of citizenship?
> If you have multiple nationalities, please choose one.

would make more sense to me. Alternatively, we could drop the hint altogether.

**Comment by SachaG** [comment]
Yes, I was planning on styling the question a bit better than that as well. The "hint" is just the description from the survey, but we can change it for the results. We might want to circle back on this once we have our "takeaways" blurbs to see how everything fits together?

**Comment by pwim** [comment]
Sure.
